testing/plot_metrics_per_site.py

```diff
@@ -0,0 +1,292 @@
+import os
```

Comment:
A short description of the script could be added.

Reply:
done here https://github.com/ivadomed/model_seg_sci/pull/87/commits/f5e097f361b6a0e63a4edb65638ea1ac510ac669